package.json

```diff
@@ -34,10 +34,11 @@
   "devDependencies": {
     "@commitlint/cli": "7.5.2",
     "@commitlint/travis-cli": "7.5.2",
-    "@silvermine/eslint-config": "2.2.0",
+    "@silvermine/eslint-config": "2.3.0",
     "coveralls": "3.0.2",
     "cz-conventional-changelog": "2.1.0",
-    "eslint": "5.11.1",
+    "espree": "^5.0.0",
+    "esutils": "^2.0.2",
```

Comment:
Could we get rid of the `^` here to lock down these dependencies?

Reply:
Sure, this is done.